theora_image_transport/src/theora_publisher.cpp

```diff
@@ -74,20 +74,13 @@ TheoraPublisher::~TheoraPublisher()
 void TheoraPublisher::advertiseImpl(
   rclcpp::Node *node,
   const std::string &base_topic,
-  uint32_t queue_size,
-  rmw_qos_profile_t custom_qos)
+  rmw_qos_profile_t custom_qos,
+  rclcpp::PublisherOptions options)
 {
   logger_ = node->get_logger();
-  // queue_size doesn't account for the 3 header packets, so we correct (with a little extra) here.
```

Comment:
I see you've removed this, is this change needed? (I don't quite know what it's doing)

Reply:
The previous comment of why that was being done doesn't make sense at all.
Also `queue_size` is not more an argument of the function.